Tools/autotest/apmrover2.py

```diff
@@ -23,7 +24,7 @@ def arm_rover(mavproxy, mav):
     wait_seconds(mav, 30)
 
     mavproxy.send('arm throttle\n')
-    mavproxy.expect('ARMED')
+    mavproxy.expect(u'ARMED')
```

Comment:
why do you need to add this? I'm wondering if there's a better way since it looks fragile since we don't run CI with python3 and it's likely to break in future.  Can we change the expect() function to do the right thing instead?  And without digging much into the issue (sorry), does this mean the terminal must be in unicode mode?